pythran/pythonic/utils/meta.hpp

```diff
@@ -14,6 +16,7 @@ template<bool V0> struct all_of<V0> {
 };
 
 template<bool... V> struct any_of;
+template<> struct any_of<> : std::integral_constant<bool, true> {};
```

Comment:
Sorry, I missed it during my first review. I think this value should be false and not true. Am I wrong?